uftrace.h

```diff
@@ -423,6 +423,8 @@ struct uftrace_record {
 	uint64_t more:   1;
 	uint64_t magic:  3;
 	uint64_t depth:  10;
+
+	/* child ip or uftrace_event_id */
 	uint64_t addr:   48;
```

Comment:
I prefer having the comment on the (right) side unless it's long.

Reply:
Ok, I changed it !